confirm/confirm.go

```diff
@@ -296,23 +291,3 @@ func Middleware(ab *authboss.Authboss) func(http.Handler) http.Handler {
 		})
 	}
 }
-
-// GenerateConfirmCreds generates pieces needed for user confirm
-// selector: hash of the first half of a 64 byte value
-// (to be stored in the database and used in SELECT query)
-// verifier: hash of the second half of a 64 byte value
-// (to be stored in database but never used in SELECT query)
-// token: the user-facing base64 encoded selector+verifier
-func GenerateConfirmCreds() (selector, verifier, token string, err error) {
```

Comment:
This is an exported function and this could break folks. Can we leave it here and deprecate?